README.md

```diff
@@ -1,24 +1,111 @@
-Instructions on setting up on your machine:
-1. Update .env file
-  - The .env file is a sensitive document which contains our bot's token, essentially a key to logging in; Slack DM Dylan Jian for access
-2. Create virtual environment by running:
-```
-  python -m venv venv
+## Welcome to the repository for the OHI/O Discord bot!
+
+Here are instructions on how to set up the repository on your own machine.
+
+Note that these instructions assume that you already have...
```

Comment:
It is possible that there I have made some further assumptions that are not listed here. If there are any parts of the README that you think would require some prerequisite knowledge that I haven't listed here, please reply to this comment.

Alternatively, if you think any of my stated assumptions are unnecessary to be written down explicitly like this, please feel free to let me know about that as well.